### PR TITLE
Migrate(npm)/migrate to npm

### PR DESCRIPTION
**Pull request by ishiasaka**
# Migrate to npm
## Cause
Fastify can't resolve when used `FastifyReply` or `FastifyRequest`, solved with change package manager to npm
## How to migration
Replace `pnpm` with `npm` in every instruction
And use npm as new package manager in this repo

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,55 +20,52 @@
     "test:cov": "jest --coverage",
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
     "test:cucumber": "cucumber-js",
-    "start:pm2": "pm2 --name Backend start pnpm -- start:prod",
+    "start:pm2": "pm2 --name Backend start dist/main.js",
     "stop:pm2": "pm2 delete 0"
```
Comment (vjumpkung) [inline package.json]: I think it should be name of the process (Backend) 
example `pm2 delete Backend`

**Review by vjumpkung — CHANGES_REQUESTED** [review]
I suggest a `npm run stop:pm2` one

**Review by namirinz — APPROVED** [review]

**Review by namirinz — DISMISSED** [review]
LGTM

**Review by Npwskp — DISMISSED** [review]
LGTM !! npm bunzai !!

**Review by PraepechPrAe — COMMENTED** [review]
Gooo

**Review by PraepechPrAe — DISMISSED** [review]
reviewed

**Pull request merged**